tox.ini

```diff
@@ -1,10 +1,12 @@
 [tox]
+
 envlist = py38, py39, py310, py311
 # isolated_build = True
 
 [testenv]
 deps =
     -r {toxinidir}/requirements_dev.txt
+    readme_renderer
```

Comment:
What is this doing?

Reply:
Thanks for calling this out. This package is intended to check if the `README.rst` can be rendered on PyPi. The inclusion of this package was inspired by [`sampleproject`](https://github.com/pypa/sampleproject/blob/fc8d83efd5749853507be959d858629d0aaf5c6e/tox.ini#L30)'s use in `tox.ini`. 

Given the current state of the package's [PyPi page](https://pypi.org/project/argparse-dataclass/) I removed it from the dependency list. With that being said we can add it later if we decide to enhance our CI processes or use automated publishing.

Reply:
Interesting, thanks.